templates/dashboard/contact_detail.html

```diff
@@ -49,8 +49,10 @@ <h1 class="h3">
 
               <textarea name="pgp"
                   {% if not current_user.is_premium() %} disabled {% endif %}
-                        class="form-control" rows=10
+                        class="form-control" rows=10 id="pgp-public-key"
                         placeholder="-----BEGIN PGP PUBLIC KEY BLOCK-----">{{ contact.pgp_public_key or "" }}</textarea>
+              <div class="text-body p-2">You can drag and drop the pgp key into the text area</div>
```

Comment:
For the styling, I'd suggest to use `alert alert-info` class as it's often used for hints. For example something like this would look ok:

<img width="1152" alt="image" src="https://user-images.githubusercontent.com/3167780/155709441-5b7a64f7-1e1e-4c90-947d-816ab6c18b9d.png">

Reply:
:+1: